lib/features/navigator/app_router.dart

```diff
@@ -29,7 +29,7 @@ class _NoTransitionRoute extends CustomRoute {
 }
 
 @AutoRouterConfig(replaceInRouteName: "View,Route")
-class AppRouter extends _$AppRouter {
+class AppRouter extends RootStackRouter {
```

Comment:
Why the change here? Just curious

Reply:
it's caused by a new version of `auto_route`